source/publishing_data/07_configuring_visualizations/02_configuring_map_view/map.rst

```diff
@@ -1,40 +1,131 @@
 
-      The map view options
+Customizing the tooltip
+-----------------------
 
-.. ifconfig:: language == 'fr'
+The "Tooltip customization" section contains a variety of options to customize the tooltips that can be displayed on the Map visualization of a dataset. A tooltip is a box that appears when clicking on an element of the map that represents data (e.g. a map marker, a geographical shape). The tooltip displays information about its related data.
 
-   .. figure:: images/map__options--fr.jpg
+.. admonition:: Important
+   :class: important
 
-      The map view options
+   For more information about tooltips and how to configure both Standard and Custom HTML tooltips, see :doc:`Configure the tooltip</publishing_data/07_configuring_visualizations/02_configuring_map_view/tooltip>` documentation.
 
+To customize the tooltips of the Map visualization, follow the instructions from the tables below.
 
-Setting up the tooltip
-----------------------
+.. list-table::
+  :header-rows: 1
 
-On a map, a click on a point will bring forward a tooltip containing more information
+  * * Option name
+    * Description
+    * Usage
+  * * Tooltip type
+    * 3 tooltip types:
 
-.. image:: images/map__tooltip--en.jpg
-        :alt: Map Tooltip
+      - Standard tooltip (default): allows the choice of a title and of the fields displayed in the tooltip.
+      - Custom HTML tooltip: allows a further customization of the tooltip, using HTML.
+      - No tooltip: disables the tooltips for the Map visualization of the dataset.
+    * 1. Click on the "Tooltip type" list box.
+      2. Choose a tooltip type.
 
-You can :doc:`configure the tooltip</publishing_data/07_configuring_visualizations/02_configuring_map_view/tooltip>` in exactly the same way than for the image gallery or the calendar.
+**Standard tooltip options**
+
+.. list-table::
+  :header-rows: 1
+
+  * * Option name
+    * Description
+    * Usage
+  * * Tooltip title
+    * Title of the tooltip based on a chosen field from the dataset.
+    * 1. Click on the "Tooltip title" list box.
+      2. Choose a field from the dataset, on which the title will be based.
+  * * Displayed fields
+    * Fields from the dataset which values will be displayed in the tooltip. In the list, the fields with a grey background are those that will be displayed in the tooltip, whereas those without a colored background will be hidden from the tooltip.
+    * To add a field in the tooltip:
+
+      1. Find the field among the list.
+      2. Click on the + icon at the right of the field.
+
+      To hide a field from the tooltip:
+
+      1. Find the field among the list.
+      2. Click on the X icon at the right of the field.
+
+      To reorder the fields:
+
+      1. Click on a field among the list.
+      2. Drag and drop the field above or below another one.
+  * * Sort in tooltips
+    * When more than one tooltip is related to one same element on the map, it is possible to define a sort to order these tooltips.
```

Comment:
I think the case where this happens is not always clear to users, especially those that never had a dataset where several points were at the same place.
How about something like "When there is more than one element at a single place on the map, clicking it will display a single tooltip where you can browse information about all these elements. It is possible to define a sort to order that information"?